frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/InsideHearingMainPage.js

```diff
@@ -163,7 +163,7 @@ const InsideHearingMainPage = () => {
     } else {
       setTranscriptText(newText);
 
-      if (Object.keys(hearing).length === 0) {
+      if (Object.keys(hearing || {}).length === 0) {
```

Comment:
the check should be `if (!hearing || Object.keys(hearing).length === 0 )`

Reply:
addressed